Tidbit/StorageAdapter/Storage/Oracle.php

```diff
@@ -79,4 +81,66 @@ protected function prepareQuery($tableName, array $installData)
 
         return $sql . ' SELECT * FROM dual';
     }
+
+    /**
+     * Patch inserted value if it look like sequence
+     *
+     * @param array $installData
+     */
+    protected function patchSequenceValues(array &$installData)
+    {
+        if (!$sequence = $this->getSequenceFromValues($installData[0])) {
+            return;
+        }
+
+        $currentValue = $this->getCurrentSequenceValue($sequence['name']);
+        $installDataCount = count($installData);
+        for ($i=0; $i <$installDataCount; $i++) {
+            $installData[$i][$sequence['field']] = $currentValue++;
+        }
+
+        $this->setNewSequenceValue($sequence['name'], $installDataCount);
+    }
+
+    /**
+     * Check array of values on containing sequence value
+     *
+     * @param array $values
+     * @return array
+     */
+    protected function getSequenceFromValues(array $values)
+    {
+        foreach ($values as $k => $v) {
+            if (substr($v, -12) == '_SEQ.NEXTVAL') {
+                return ['field' => $k, 'name' => substr($v, 0, -8)];
+            }
+        }
+        return [];
+    }
+
+    /**
+     * rtfn
+     *
+     * @param string $sequenceName
+     * @return int
+     */
+    protected function getCurrentSequenceValue($sequenceName)
+    {
+        $sql = sprintf("SELECT %s.NEXTVAL as current_val FROM dual", $sequenceName);
```

Comment:
SEQ_NAME.NEXTVAL will increase sequence value, I guess select from user_sequences table will be preferable for this function:
SELECT last_number FROM user_sequences WHERE sequence_name='SEQ_NAME';